custom/icds_reports/migrations/0172_add_ward_columns.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated on 2020-03-21
+from __future__ import unicode_literals
+from django.db import migrations
+from custom.icds_reports.utils.migrations import get_view_migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0171_add_sdr_field_and_view'),
+    ]
+
+    operations = [
+        migrations.RunSQL('ALTER TABLE awc_location ADD COLUMN awc_ward_1 VARCHAR'),
```

Comment:
these should be `TEXT` not `VARCHAR`

Reply:
@calellowitz  updated the type, Does it looks good now?